base/setup/usetup/bootsup.c

```diff
@@ -2378,9 +2405,11 @@ InstallFatBootcodeToPartition(
         /* Update 'boot.ini' */
         CombinePaths(DstPath, ARRAYSIZE(DstPath), 2, SystemRootPath->Buffer, L"boot.ini");
 
+        GetBootSectFilePath(szBootSectFile, ARRAYSIZE(szBootSectFile));
+
         DPRINT1("Update 'boot.ini': %S\n", DstPath);
         Status = UpdateBootIni(DstPath,
-                               L"C:\\bootsect.ros",
+                               szBootSectFile,
```

Comment:
No, this is wrong. The correct letter is really "C:", this is what is required by NTLDR when booting a bootsector, because "C:" is also the name of the boot drive, whatever it can be.